common/config.js

```diff
@@ -0,0 +1,36 @@
+(function() {
+    'use strict';
+
+    angular.module('chaise.config', ['ermrestjs'])
```

Comment:
technically now this module relies on `utils.js`, so maybe we should list it here.

Reply:
That's true. I'll add it to the dependency list